sys/net/gnrc/network_layer/sixlowpan/frag/vrb/gnrc_sixlowpan_frag_vrb.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#include "net/ieee802154.h"
+#include "xtimer.h"
+
+#include "net/gnrc/sixlowpan/frag/vrb.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+static gnrc_sixlowpan_frag_vrb_t _vrb[GNRC_SIXLOWPAN_FRAG_VRB_SIZE];
+static char l2addr_str[3 * IEEE802154_LONG_ADDRESS_LEN];
```

Comment:
this is only used for debugging, so better `#if ENABLE_DEBUG` around it?

Reply:
Since https://github.com/RIOT-OS/RIOT/pull/5166 is merged this is not possible anymore. However, `l2addr_str` should be kicked by the optimizer, as it is only used in the `DEBUG` macro.